raster/src/main/scala/geotrellis/raster/render/ColorMethods.scala

```diff
@@ -48,14 +48,13 @@ trait MultibandColorMethods extends MethodExtensions[MultibandTile] {
 
     if(self.bandCount == 3) {
       self.convert(IntConstantNoDataCellType).combine(0, 1, 2) { (rBand, gBand, bBand) =>
-        val r = if (isData(rBand)) { rBand } else 0
-        val g = if (isData(gBand)) { gBand } else 0
-        val b = if (isData(bBand)) { bBand } else 0
+        var transparent = true
+        val r = if (isData(rBand)) { transparent = false; rBand } else 0
+        val g = if (isData(gBand)) { transparent = false; gBand } else 0
+        val b = if (isData(bBand)) { transparent = false; bBand } else 0
 
-        if(r + g + b == 0) 0
-        else {
-          ((r & 0xFF) << 24) | ((g & 0xFF) << 16) | ((b & 0xFF) << 8) | 0xFF
-        }
+        if(transparent) 0
+        else ((r & 0xFF) << 24) | ((g & 0xFF) << 16) | ((b & 0xFF) << 8) | 0xFF
```

Comment:
I wanted to make this "transparency" detection more cheap, since this function is applied to all pixels of the raster.